interface/patient_file/summary/demographics.php

```diff
@@ -164,16 +164,15 @@ function image_widget($doc_id, $doc_catg)
 
 <head>
 
-    <?php Header::setupHeader(['common']); ?>
-    <link rel="stylesheet" type="text/css" href="../../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
-    <script type="text/javascript" src="../../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+    <?php Header::setupHeader(['common', 'opener']); ?>
```

Comment:
Is opener needed here. I though it was meant for popup scripts in tabs?(I could be wrong)